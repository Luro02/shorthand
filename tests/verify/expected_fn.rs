use shorthand::ShortHand;

#[derive(ShortHand)]
#[shorthand(verify(ffn = "Self::verify_field"))]
struct Example {
    field: usize,
}

fn main() {}
