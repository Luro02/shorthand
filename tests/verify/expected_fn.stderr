error: expected `fn`
 --> $DIR/expected_fn.rs:4:20
  |
4 | #[shorthand(verify(ffn = "Self::verify_field"))]
  |                    ^^^
